### DISCUSSION TITLE
How to avoid caching with getServerSideProps

### DISCUSSION THREAD

**Discussion by bugrakocak** [vercel/next.js#21619] [Help] [answered]
Hi.

I deployed my next.js app to Heroku and I use getServerSideProps to get my data from a custom CMS API but when the content is updated or newly created the front page doesn't show updated content. It only becomes visible after a while like an hour or more. How can I avoid or configure this problem?

Note: I am using `apolloClient.query` inside of getServerSideProps but I don't know how this is relevant to the current situation.

Thank you.

**Comment by bugrakocak** [comment, marked as answer]
In case of anyone face with this problem; it wasn't about next.js, I created a single instance of apollo client.

**Reply by KenneyE** [reply]
Are you able to elaborate on this answer? I am running into this same issue and it sounds like I have a very similar configuration as you. Some more details on your fix would be greatly appreciated!
